libpromises/evalfunction.c

```diff
@@ -238,8 +238,8 @@ static JsonElement* VarRefValueToJson(EvalContext *ctx, const FnCall *fp, const
             break;
 
         case RVAL_TYPE_CONTAINER:
-            convert = value;
-            *allocated = false;
+            convert = JsonCopy(value);
```

Comment:
To make clear what I mean, the `value` variable is const and can't be changed (and that's correct, `value` should be const) but it is assigned to `convert` which is not const.